packages/apps/escrow-dashboard/package.json

```diff
@@ -28,7 +28,7 @@
     "react-redux": "^8.0.5",
     "react-router-dom": "^6.4.3",
     "react-test-renderer": "^18.2.0",
-    "recharts": "^2.1.16",
+    "recharts": "2.9.0",
```

Comment:
```suggestion
    "recharts": "^2.9.0",
```

Reply:
@leric7 We have to force recharts to version 2.9.0. I tested it with a higher one and escrow-dashboard tests fail